internal/dataplane/parser/translators/httproute.go

```diff
@@ -369,8 +405,18 @@ func GeneratePluginsFromHTTPRouteFilters(filters []gatewayapi.HTTPRouteFilter, p
 		case gatewayapi.HTTPRouteFilterResponseHeaderModifier:
 			kongPlugins = append(kongPlugins, generateResponseHeaderModifierKongPlugin(filter.ResponseHeaderModifier))
 
-		case gatewayapi.HTTPRouteFilterExtensionRef,
-			gatewayapi.HTTPRouteFilterRequestMirror,
+		case gatewayapi.HTTPRouteFilterExtensionRef:
+			plugin, err := generateExtensionRefKongPlugin(filter.ExtensionRef)
+			if err != nil {
+				return nil, "", err
+			}
+			if len(pluginsAnnotation) > 0 {
+				pluginsAnnotation = fmt.Sprintf("%s,%s", pluginsAnnotation, plugin)
```

Comment:
Nothing critical but using https://pkg.go.dev/strings#Builder should be more effective when appending to a string.